src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/command.html.twig

```diff
@@ -0,0 +1,249 @@
+{% extends '@WebProfiler/Profiler/layout.html.twig' %}
+
+{% block menu %}
+    <span class="label">
+        <span class="icon">{{ source('@WebProfiler/Icon/command.svg') }}</span>
+        <strong>CLI / Command</strong>
```

Comment:
Commands is enough no? otherwise it feels redondant 2 times "command"

Reply:
I've renamed it "Console Command" consistently with the header listing "Console Commands".

Reply:
I've renamed it "Console Command" consistently with the header listing "Console Commands".